assets/sass/patterns/molecules/site-header-nav-bar-primary.scss

```diff
@@ -98,6 +147,16 @@
   }
 }
 
+@media only all and (max-width: 730px) {
+  .nav-primary__item--first {
+    display: flex !important;
```

Comment:
Please remove that !important, you can add more the classes to override the displayed style. Instead of just ".nav-primary__item--first" you can use ".nav-primary .nav-primary__item--first"

Reply:
I removed that !important